resource_data_model.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE "resources" (
+  "resourceId" integer PRIMARY KEY,
+  "title" varchar(255) NOT NULL,
+  "tags" varchar(255)[],
```

Comment:
tags will be redundant between the resources, create a tag table (id, tag_name) and store the reference id here